doc/man1/rand.pod

```diff
@@ -18,12 +18,14 @@ I<num>
 
 =head1 DESCRIPTION
 
-The B<rand> command outputs I<num> pseudo-random bytes after seeding
-the random number generator once.  As in other B<openssl> command
-line tools, PRNG seeding uses the file I<$HOME/>B<.rnd> or B<.rnd>
-in addition to the files given in the B<-rand> option.  A new
-I<$HOME>/B<.rnd> or B<.rnd> file will be written back if enough
-seeding was obtained from these sources.
+This command generates I<num> random bytes using a cryptographically
+secure pseudo random number generator (CSPRNG).
+
+The random bytes are generated using the L<RAND_bytes(3)> function,
+which provides a security level of 256 bits, provided it managed to
+seed itself successfully from a trusted operating system entropy source.
+Otherwise, the command will fail with a nonzero error code.
+For more details, see L<RAND_bytes(3)>, L<RAND(7)>, and L<RAND_DRBG(7)>.
```

Comment:
> I hope that return 1 still means success. If not ensure backward compatibility code first. With other words what is non-zero error code?

Oh, thank you Roumen for pointing that out! That's obviously a documentation error. I will fix it here right away and create a separate pull request for master, sine the original pr was already merged.

Reply:
Correction, forget my last comment: this manual page is talking about the `openssl rand` command, not about `RAND_bytes()`, etc.  Like all Unix programs, it returns zero on success and a non-zero return value in the case of an error.